snuba/datasets/entities/transactions.py

```diff
@@ -86,20 +98,89 @@
 )
 
 
+class SampledSimplePipelineBuilder(SimplePipelineBuilder):
+    def build_planner(
```

Comment:
Isn't the `build_execution_pipeline` function still getting the old sample rate from the request?

Reply:
Just saw that `build_execution_pipeline` calls `build_planner()`. Ignore this.